community/contributingGuides/Contributing.asciidoc

```diff
@@ -15,15 +15,15 @@ As an open source project we are always on the lookout for community members tha
 
 === Apache License
 
-All contributions to AeroGear are made under the http://apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0].
+All contributions to AeroGear are made under the http://apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0^].
 
 === Help Others
-Have experience with mobile or HTML5 based technologies? Interested in sharing your knowledge?
+Have experience with mobile, container and orchestration based technologies? Interested in sharing your knowledge?
 
-If you've got ideas for the project, or want to discuss ways to improve AeroGear, join the https://lists.jboss.org/mailman/listinfo/aerogear-dev[aerogear-dev] mailing list] and share what your working on, or struggling with. For more immediate gratification join the IRC channel (#aerogear). There are are usually a good bunch of people hanging out there. Ask your questions, share your experience, or just make jokes :-)
+If you've got ideas for the project, or want to discuss ways to improve AeroGear, join the https://groups.google.com/forum/#!forum/aerogear[aerogear-dev^] google group and share what your working on, or struggling with. For more immediate gratification join https://webchat.freenode.net/?channels=%23aerogear[#aerogear @ irc.freenode.net^]. There are are usually a good bunch of people hanging out there. Ask your questions, share your experience, or just make jokes :-)
 
 ==== Found An Issue?
-Have you run into an issue with one of our examples, sites, or have a suggestion? The best place to start is probably talking about it in the forums or IRC. If you've already done that, or just want to create the issue please do at https://issues.jboss.org/browse/AEROGEAR.
+Have you run into an issue with one of our https://docs.aerogear.org/aerogear/latest/con_services.html[services], https://github.com/aerogear?utf8=%E2%9C%93&q=showcase&type=&language=[apps^], or link:../../../sdks[sdks] or just have a suggestion? The best place to start is probably talking about it in the google group or IRC. If you've already done that, or just want to create the issue please do in the https://issues.jboss.org/browse/AEROGEAR[AEROGEAR project on JIRA^].
```

Comment:
We could also mention that they are welcome to open Github issues.

Reply:
@pb82 will do